iModelCore/ECDb/Tests/NonPublished/SchemaUpgradeTests.cpp

```diff
@@ -10622,6 +10622,82 @@ TEST_F(SchemaUpgradeTestFixture, DeleteKindOfQuantityFromECSchema)
     AssertSchemaUpdate(editedSchemaXml, filePath, {false, false}, "Deleting KindOfQuantity from an ECSchema");
     }
 
+TEST_F(SchemaUpgradeTestFixture, DeleteKoQWithMajorSchemaChangeShouldPass)
+    {
+    SchemaItem schemaItem(R"xml(
+        <?xml version='1.0' encoding='utf-8'?>
+        <ECSchema schemaName='TestSchema' alias='ts' version='1.0.0' xmlns='http://www.bentley.com/schemas/Bentley.ECXML.3.1'>

```

Comment:
Since this is a new test use an EC 3.2 schema

Reply:
Done !